resources/qml/voip/ScreenShare.qml

```diff
@@ -14,8 +14,15 @@ Popup {
     anchors.centerIn: parent;
 
     Component.onCompleted: {
+        if (CallManager.screenShareX11Available)
```

Comment:
Shouldn't this switch be the other way around? On Wayland you can use both for XWayland windows, but I don't think you can use both for wayland windows. So on Wayland you should always prefer the portal one, I would assume. And tbh, I am not sure the frontend code should do that. Just set an internal default in the backend or so?

Reply:
I now set the default in CallManager.cpp: It is available with XDP if the GStreamer PipeWire plugin is available and with X11 if DISPLAY is set. The former is the default if Qt runs with Wayland.